dolphinscheduler-task-plugin/dolphinscheduler-task-http/src/test/java/org/apache/dolphinscheduler/plugin/task/http/HttpTaskTest.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.plugin.task.http;
+
+import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.EXIT_CODE_FAILURE;
+import static org.apache.dolphinscheduler.plugin.task.api.TaskConstants.EXIT_CODE_SUCCESS;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import java.io.IOException;
+import java.util.*;
```

Comment:
Please avoid importing *

Reply:
Thanks, i will fix it

Reply:
> Please avoid importing *

I'm confuse why our spotless doesn't detect this. The previous checkstyle can be checked correctly.

Reply:
> > Please avoid importing *
> 
> I'm confuse why our spotless doesn't detect this. The previous checkstyle can be checked correctly.

@SbloodyS I will look into it and give a quick fix for this.

Reply:
@EricGao888 I have fixed this problem, please review again.

Reply:
BTW, can we add `remove unused import` in spotless

Reply:
> BTW, can we add `remove unused import` in spotless

@ruanwenjun Good idea! Actually I'm already working on the PR. Also we could add some extra rules for import order. Will submit a PR later tonight : )